server/sonar-web/src/main/js/apps/quality-gates/views/gate-projects-view.js

```diff
@@ -38,9 +38,9 @@ export default Marionette.ItemView.extend({
       format (item) {
         return item.name;
       },
-      searchUrl: '/api/qualitygates/search?gateId=' + qualityGate.id,
-      selectUrl: '/api/qualitygates/select',
-      deselectUrl: '/api/qualitygates/deselect',
+      searchUrl: window.baseUrl + '/api/qualitygates/search?gateId=' + qualityGate.id,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)